### ISSUE TITLE
Add ability to to collapse/fold long outputs

### ISSUE THREAD

**Issue by cameronfreer** [OpenGen/GenSQL.publish#9]
Some cells have very long outputs which take a lot of time to scroll through. It would be convenient to be able to click on some thin widget to the left of any part of the output to collapse/fold the output to nothing or to a few lines -- like in jupyterlab and jupyter notebook cells.

**Comment by cameronfreer** [comment]
Correct, notebook cells (in the sense of #5 or #8, not #7) which each have
an "Execute" button.

- Cameron

**Comment by Schaechtle** [comment]
An alternative could be that output has maximum height and is scrollable.
